rad/src/operators/array.rs

```diff
@@ -58,6 +62,44 @@ pub fn map(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
     Ok(RadonArray::from(result).into())
 }
 
+pub fn sort(input: &RadonArray) -> Result<RadonArray, RadError> {
+    // Sort not applicable if not homogeneous
+    if !input.is_homogeneous() {
+        Err(RadError::UnsupportedOpNonHomogeneous)?;
+    }
+    // Get discriminant and values
+    let disc = input.inner_type();
+    let value = input.value();
+
+    let mut result = Vec::new();
+    // Distinguish each discriminant case
+    if disc == discriminant(&RadonTypes::from(RadonString::default())) {
+        result = value
+            .iter()
+            .cloned()
+            .sorted_by(|a, b| match (a, b) {
+                (RadonTypes::String(a), RadonTypes::String(b)) => a.cmp(&b),
+                _ => unreachable!(),
+            })
+            .collect();
+    } else if disc == discriminant(&RadonTypes::from(RadonInteger::default())) {
+        result = value
+            .iter()
+            .cloned()
+            .sorted_by(|a, b| match (a, b) {
+                (RadonTypes::Integer(a), RadonTypes::Integer(b)) => a.cmp(&b),
+                _ => unreachable!(),
+            })
+            .collect();
+    } else {
+        Err(RadError::UnsupportedSortOp {
+            inner_type: value[0].clone().radon_type_name(),
```

Comment:
Maybe this will panic if `value` is empty? Add a test for sorting an empty array, it should return an empty array.